app/routes/overview/components/PublicFrontpage.css

```diff
@@ -65,3 +65,8 @@
 .linkDescription {
   margin-left: 30px;
 }
+
+.latestReadme {
+  min-height: 442px;
+  width: 330px;
```

Comment:
Why not just use percentages

Reply:
I guess I can use percentage for the width, but I still need some fixed size for the height, otherwise you'd end up with layout shifting when loading.

Reply:
Although, with `witdh: 100%` the image is going to be super huge for "middle" sized screens where you still have one column. Not super likely, but it is a drawback. Kinda feel the best might be to just set the size fixed and you shouldn't have issues like this.

Reply:
I changed this to `max-width`, `min-width` and `width: 100%`. That should cover all the points for responsiveness, no layout shift and reasonably sized image.